src/components/Button.tsx

```diff
@@ -28,6 +28,7 @@ export enum BtnSizes {
 export interface ButtonProps {
   onPress: () => void
   style?: StyleProp<ViewStyle>
+  buttonStyle?: StyleProp<ViewStyle>
```

Comment:
I wonder if we should introduce another prop for the icon placement to avoid exposing too much of the inner layout, and avoiding incorrect usages.

Reply:
yeah i also felt this was hacky. i updated to use a boolean prop, maybe you want to take another look at the changes to the Button?

Reply:
Yes looks good, we can remove `buttonStyle` then 👍

Reply:
removed!